backend/utils/message_utils.py

```diff
@@ -81,7 +86,7 @@ async def get_room_messages(
 
 
 async def get_message(
-    org_id: str, room_id: str, message_id: str
+    org_id: str, room_id: str, message_id: str, page: int, limit: int
 ) -> Optional[dict[str, Any]]:
     """Get a specific message in a room.
```

Comment:
pagination is not needed in this function. since it gets a single message.

Reply:
Oh right, I'll remove it.